siloLapisTests/test/aggregatedQueries/noMutationAt23270.json

```diff
@@ -1,11 +1,11 @@
 {
   "testCaseName": "no mutation at 23063",
   "lapisRequest": {
-    "nucleotideMutations": "23063."
+    "nucleotideMutations": ["230."]
```

Comment:
The test case name and the file name are not in sync with the query anymore.

I'd also like to suggest a second test case for "no mutation at 230" which is to query for `nucleotideMutations: [230C]`, it should return the same number.

Reply:
Do you mean an explicit query for "is equal to the symbol in the reference genome"? I think I'd leave that for SILO, as that logic is handled there.